templates/keyword_result.html

```diff
@@ -0,0 +1,30 @@
+{{ define "content" }}
+
+<div class="result-keyword">
```

Comment:
How about making it `<section>`

```suggestion
<section class="result-keyword">
```

So that below we canuse the header tag:

```
<header class="result-keyword__header">
```

Reply:
Nice idea. Fixed in 60a0a30250d5fd6437c773d0514e0347087250d5 d9ad685bde7b9780139fa31e19ff43772b488207